brahmapsen/defi-assetmgr
# Feat/portis

This is the portis integration. Some notes:
1. I wanted to style the button purple (since that is the Portis's theme color), but I don't actually know how to do that.
2. I had to delete package-lock.json and reinstall in order to get it working in my environment. Let me know if that breaks anything for you all.

How to test:
1. First, create a Portis wallet at portis.io
2. Set the environment variables as explained in .env.example (sensitive information I have sent in the discord).
3. Start the ganache chain on using ganache cli like so (this sets your chain ID to 999):
`ganache-cli --fork https://mainnet.infura.io/v3/<YOUR_API_KEY> --i 999 --chainId 999`
4. Deploy the contract to ganache
5. In your Portis wallet, go to settings, import wallet, and import one of the test wallets.
6. Sign in with portis and try to use the dapp normally. When you request a portfolio, the Portis interface will say that the transaction value is 0 (it only shows mainnet transaction values), but the dapp should show that there is coin in your account.
